src/main/kotlin/com/terraformation/backend/accelerator/model/CohortModel.kt

```diff
@@ -11,7 +11,7 @@ import org.jooq.Record
 data class CohortModel<ID : CohortId?>(
     val id: ID,
     val name: String,
-    val participantIds: List<ParticipantId>,
+    val participantIds: List<ParticipantId>?,
```

Comment:
When will this be null as opposed to an empty list? I think it may be slightly preferable for the model class to always have this non-nullable just so the code that deals with models doesn't have to handle nulls.

Reply:
Good call, I will update this in #1680